### DISCUSSION TITLE
posts/programming/2023yearreview

### DISCUSSION THREAD

**Discussion by giscus** [movie42/hyunsu#10] [General]
# posts/programming/2023yearreview

페이지를 찾을 수 없습니다.

https://www.hyunsu.info/posts/programming/2023yearreview

**Comment by gitjoyoung** [comment]
글을 정말 재미있게 잘쓰시네요 b

**Reply by movie42** [reply]
뭔가 기술적으로 도움되는건 아니지만 그래도 읽어주셔서 감사합니다. 🥹

**Reply by movie42** [reply]
테스트

**Comment by hotbreakb** [comment]
방금 <프로그래머의 뇌> 도서관에서 예약했어요 📚

**Reply by movie42** [reply]
저는 아직 장바구니에만 있네요. 읽고 있는 책 후다닥 읽고 저도 시작 하렵니다
